daemon/daemon.go

```diff
@@ -839,7 +839,7 @@ func NewDaemonFromDirectory(config *Config, eng *engine.Engine, registryService
 		}
 	})
 
-	if config.EnableSelinuxSupport {
+	if config.PlatformSpecific.EnableSelinuxSupport {
```

Comment:
Almost certainly not - but with the current PRs, the Windows daemon is a very long way from compiling on this code base. This is tagged for a later PR on my list already (and doesn't affect the client)